urllib3/connectionpool.py

```diff
@@ -652,7 +652,15 @@ def urlopen(self, method, url, body=None, headers=None, retries=None,
 
         # Check if we should retry the HTTP response.
         if retries.is_forced_retry(method, status_code=response.status):
-            retries = retries.increment(method, url, response=response, _pool=self)
+            try:
+                retries = retries.increment(method, url, response=response, _pool=self)
```

Comment:
Hmm looks like we're calling retries.increment(...) an extra time with this change than before. Was this a bug before where we weren't incrementing during retries, or is it a bug now that we're double-incrementing?